andela/ah-backend-tesseract
# #159951971- implement user's following/unfollowing

#### What does this PR do?
- enables users to follow and unfollow each other
#### Description of Task to be completed?
- ensure users can follow other users' profile
- implement unfollowing a user
#### How should this be manually tested?
- ensure that you have at least two users signed up.
- follow a user by accessing   `api/profiles/<str:username>/follow`  `POST`
-unfollow a user by accessing  `api/profiles/<str:username>/follow`  `DELETE`
- Modies retrieve users profile to return followers.

#### What are the relevant pivotal tracker stories? 
#159951971

#### Screenshots
follow a user
<img width="715" alt="screen shot 2018-09-17 at 21 30 36" src="https://user-images.githubusercontent.com/30200454/45642555-337bb580-bac1-11e8-8ccd-a5f9b9b55595.png">
Retrieve users profile
<img width="730" alt="screen shot 2018-09-17 at 21 31 14" src="https://user-images.githubusercontent.com/30200454/45642653-7ccc0500-bac1-11e8-92b0-f74b4ac1b299.png">
